### DISCUSSION TITLE
casual-edebug?

### DISCUSSION THREAD

**Discussion by bramadams** [kickingvegas/casual-suite#43] [Ideas]
Hi,

Just starting to explore the casual-xyz packages, based on the System Crafters stream, very interesting ideas!

Are there any plans for a casual-edebug package? edebug is a must for debugging lisp code, but there are so many (often hidden) features, from (un)setting breakpoints, to navigation, watchpoints, backtraces or even changing focus/keeping track of the most relevant buffer.

**Comment by kickingvegas** [comment]
@bramadams It’s on the roadmap. FWIW, I’ve made an early prototype earlier this summer where my initial impression is that while a Transient menu helps in using Edebug, it’s not a complete solution to smoothing all the sharp/wonky edges to it. 
Perhaps in releasing Casual Edebug, others will feel motivated to improving it.

**Reply by bramadams** [reply]
Definitely, looking forward to it!
